packages/inspection-report/src/components/DamageReport/DamageReport.js

```diff
@@ -101,6 +101,7 @@ export default function DamageReport({
   inspectionId,
   vehicleType,
   damageMode,
+  currencyCharacter,
```

Comment:
didnt you forgot to add the prop to the proptypes? If so, make it optional with the "$" as a default value

Reply:
Resolved all the comments